### PR TITLE
Rework brackets

### PR DESCRIPTION
**Pull request by bhuga**
Potential solution to #54.

### CODE REVIEW

**Comment by haacked** [comment]
Looks good to me. @benbalter any concerns?

**Comment by benbalter** [comment]
@bhuga It looks like the variable delimiter was changed in babcaa2a18043ea74aec05b2424fa6c469ee7115. Did you also change things on the .com side as well?

**Comment by benbalter** [comment]
Also need to mirror the content filter in [`_includes/sidebar.html`](https://github.com/github/choosealicense.com/blob/gh-pages/_includes/sidebar.html#L7)

**Comment by bhuga** [comment]
@benbalter I took it out so that I wouldn't have to change things on the .com side, and then ran into the variable/URL collision noted earlier. This is trying to bring the license files and .com into sync with no bugs.

I actually just removed the content filter in the sidebar. the `how:` fields of the YAML front matter don't have anything to replace.

**Comment by benbalter** [comment]
As long as things don't :boom: when we re-vendor thing into .com (which we should do following the merge), :+1:.

Thanks for taking the time to do the deep dive on this one. Not the most exciting problem to tackle, but important to get right and will make things a lot easier going forward.

**Comment by bhuga** [comment]
Yeah, little things like `<<url>>` on .com and missing FSF URLS here are hard to miss. Hopefully with this convention in place we'll be good for easy-peasy `cp licenses/*` in the future. Going to go ahead and merge this and I will try to get the .com pull updated today too.

**Comment by gjtorikian** [comment]
Is there a list somewhere of all potential `{ }` variables? I'm doing some [lazy `gsub`ing right now](https://github.com/gjtorikian/addalicense.com/blob/master/server.rb#L92-L94) but I see new variables like `name of copyright owner` in this PR (which should probably just be `fullname` or something, no?).

**Comment by bhuga** [comment]
@gjtorikian you would change to subbing on `[variable]` after this pull. `{variable}` is no longer meant to be interpolated.

I just found a few more cases where we were interpolating variables which were wrong. The GPL actually has an inducement to use itself at the end that I fixed, and I changed the artistic license to be correctly filled out on repo creation. Quick :eyes: on those changes are :cool:

**Comment by benbalter** [comment]
Could punt to a different PR, but would be awesome to document A) what brackets we interpolate, and B) what variables we replace on the .com side, perhaps in the "adding a license" section of the readme so we don't have to rehash this again.

**Comment by bhuga** [comment]
I'll do the README here. This was already one pull, let's do it right. Need to run just now though.

**Comment by bhuga** [comment]
Okay, README is done. Any more thoughts?

**Comment by benbalter** [comment]
:sparkles: :shipit:

File: _includes/sidebar.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <div class='how-to-apply callout'>
       <h5>How to apply this license</h5>
       <p>
-        {{ page.how | replace:"<<","[" | replace:">>","]" }}
+        {{ page.how  }}
```
Comment (haacked) [inline _includes/sidebar.html:7]: Nitpick nanny here: There's 2 spaces after "how" there. :trollface:

Comment (bhuga) [inline _includes/sidebar.html:7]: Fix pushed. Thanks for noticing. :dart:

**Comment by gjtorikian** [comment]
What does .com do when a repo doesn't have, for example, a description? Replace the variable with an empty string?

**Comment by bhuga** [comment]
Should be. It's actually whatever mustache does.

**Pull request merged**